30/hw-30.py

```diff
@@ -0,0 +1,40 @@
+import random
+import requests
+
+
+# Task 1. ====================================
+sites = [
+        'https://www.google.com/',
+        'https://www.facebook.com/',
+        'https://twitter.com/',
+        'https://www.amazon.com/',
+        'https://www.apple.com/'
+        ]
+
+def get_random_site(lst):
+        return random.choice(lst)
+
+res = requests.get(get_random_site(sites))
+print(f'Status code: {res.status_code}, url: {res.url}, HTML length: {len(res.text)}')
+
+
+# Task 2. ====================================
+try:
+    city = input('Enter your city: ')
+    coords_url = 'https://geocoding-api.open-meteo.com/v1/search'
+    city_coords = requests.get(f'{coords_url}?name={city}&count=1')
+    data = city_coords.json()
```

Comment:
nit: better to create functions for all logic parts like:
`get_city_coordinates`

Reply:
Fixed